mesa/experimental/jupyter_viz.py

```diff
@@ -281,7 +221,17 @@ def portray(g):
             out["c"] = c
         return out
 
-    space_ax.scatter(**portray(space))
+
+    # figure(figsize=(50, 50), dpi=80)
+    space_fig = Figure()
+    space_ax = space_fig.subplots()
```

Comment:
This function already has `space_ax` in the argument, so these 2 lines are unnecessary.

Reply:
Upon reading further, the whole new change from this line to L234 are duplicate of `make_space`. Please remove it.

Reply:
introduced by automated merge, should be resolved